app/main.py

```diff
@@ -1,3 +1,42 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> object:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: int | float) -> object:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: int | float) -> object:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int | float) -> object:
+        return Distance(round((self.km / other), 2))
+
+    def __lt__(self, other: int | float) -> object:
+        return self.km < other
+
+    def __gt__(self, other: int | float) -> object:
```

Comment:
The return type annotation `-> object` is not ideal. According to the checklist, you should use a more specific type annotation like `-> bool` for better clarity and type safety.